docs/manual-doc/statement-syntax/data-definition-statements/create-tag-edge-syntax.md

```diff
@@ -0,0 +1,87 @@
+### Create TAG / EDGE Syntax
+
+```
+CREATE {TAG | EDGE} tag_name|edge_name
+    (create_definition, ...)
+    [tag_edge_options]
+  
+create_definition:
+    prop_name data_type
+    
+tag_edge_options:
+    option [, option ...]
+
+option:
+    TTL_DURATION [=] ttl_duration
+    | TTL_COL [=] prop_name
+```
+
+Nebula's graph schema is composed of tags and edges, either of which may have properties. `CREATE TAG` statement defines a tag with the given name. `CREATE EDGE` statement is to define an edge type.
+
+There are several aspects to this syntax, described under the following topics in this section:
+
+* [Tag Name and Edge Type Name](#tag-name-and-edgetype-name)
+
+* [Property Name and Data Type](#property-name-and-data-type)
+
+### Tag Name and EdgeType Name
+
+* **tag_name and edge_name**
+
+    The name of tags and edgeTypes should be **unique** within the space. Once the name is defined, it can not be altered. The rules of tag and edgeType names are the same as those for names of spaces. See [Schema Object Name](../../language-structure/schema-object-names.md)
+
+### Property Name and Data Type
+
+* **prop_name**
+
+    prop_name indicates the name of properties. It must be unique for each tag or edgeType.
+
+* **data_type**
+
+    data_type represents the data type of each property. For more information about data types that Nebula Graph supports, see data types section.
+    
+    > NULL and NOT NULL constrain are not supported yet when creating tags/edges (comparing with relational databases).
+
+### Time-to-Live (TTL) syntax
+
+* TTL_DURATION
+
+    ttl_duration specifies the life cycle of vertices (or edges). TTL expires the vertexes or edges after the specified number of seconds has passed since the TTL_COL configured value; i.e. the expiration threshold is TTL_COL configured property's value plus the specified number of seconds.
```

Comment:
> TTL expires the vertexes or edges after the specified number of seconds has passed since the TTL_COL configured value;

The ttl part is only configuration which means that it won't work after the mechanism being triggered.

Reply:
I suggest:
The vertexes or edges would be expired after the given **ttl_duration** once the ttl mechanism being triggered.